plugins/source/aws/resources/services/frauddetector/rules_fetch.go

```diff
@@ -0,0 +1,32 @@
+package frauddetector
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/frauddetector"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func fetchFrauddetectorRules(ctx context.Context, meta schema.ClientMeta, _ *schema.Resource, res chan<- interface{}) error {
+	c := meta.(*client.Client)
+	svc := c.Services().FraudDetector
+
+	input := new(frauddetector.GetRulesInput)
```

Comment:
I think `DetectorId` [is a required field](https://docs.aws.amazon.com/sdk-for-go/api/service/frauddetector/#GetRulesInput) for `GetRulesInput`, so this will have to be made into a child relationship with the parent detector's ID used for this call.

Reply:
Nice catch

Reply:
done